run.sh

```diff
@@ -92,17 +98,41 @@ run() {
 case "$1" in
   gc) # C-based GTK demo
     setup_deps
-    check_gtk4
     build_gtk_wasm
     cd gtk-c
     build_container
     build_host $(pkg-config --cflags --libs gtk4)
     run
     ;;
 
-  gr) # Rust-based GTK demo; uses wasm modules from gtk-c
+  grc) # Rust-based GTK demo; uses wasm modules from gtk-c
+    setup_deps
+    (
+      cd gtk-rust && \
+        cargo build
+    )
     build_gtk_wasm
     cp -uv gtk-c/{hunter.wasm,runner.wasm} gtk-rust/
+    (
+      cd gtk-rust && \
+        cargo run
+    )
+    ;;
+
+  gcr) # C-based GTK demo with Rust wasm modules
+    setup_deps
+    (
+      cd gtk-rust
+      cargo build
+    )
+    cp -uv gtk-rust/{hunter.wasm,runner.wasm} gtk-c/
+    cd gtk-c
+    build_host $(pkg-config --cflags --libs gtk4)
+    run
+    ;;
+
+  gr) # Rust-based GTK demo; uses wasm modules from gtk-rust
```

Comment:
Are you intending to add other files? There are no rust wasm modules (yet? I wasn't intending to add them, since the rust host/container suffices for the demo, I think).

Reply:
Definitely :) I'll add WIP to the title though